en/docs/testing/test-rest-endpoints-via-the-openapi-console.md

```diff
@@ -0,0 +1,32 @@
+# Test REST Endpoints via the OpenAPI Console
+
+Devant offers an integrated OpenAPI Console to test REST endpoints for the Service components you create and deploy. Since Devant secures REST APIs with OAuth 2.0 authentication, the OpenAPI Console generates test keys to help you test your APIs.
```

Comment:
```suggestion
Devant provides an integrated OpenAPI Console for testing REST endpoints of your deployed integrations. Since Devant secures REST APIs with OAuth 2.0 authentication, the console automatically generates test keys to facilitate API testing.
```